tools/odbc/statement_functions.cpp

```diff
@@ -189,7 +189,10 @@ template <class CAST_OP, typename TARGET_TYPE, class CAST_FUNC = std::function<t
 static bool CastTimestampValue(duckdb::OdbcHandleStmt *hstmt, const duckdb::Value &val, TARGET_TYPE &target,
                                CAST_FUNC cast_timestamp_fun) {
 	try {
-		timestamp_t timestamp = cast_timestamp_fun(val.GetValue<int64_t>());
+		auto timestamp = timestamp_t(val.GetValue<int64_t>());
```

Comment:
@maiadegraaf Could you write tests for this in ODBC?
I could add a FIXME that can be addressed later if that works for you?

Reply:
Sure, a FIXME works for me :)